baseplate/clients/redis.py

```diff
@@ -1,24 +1,54 @@
 from math import ceil
+from time import perf_counter
 from typing import Any
 from typing import Dict
 from typing import Optional
 
 import redis
 
-from prometheus_client import Gauge
-
 # redis.client.StrictPipeline was renamed to redis.client.Pipeline in version 3.0
 try:
     from redis.client import StrictPipeline as Pipeline  # type: ignore
 except ImportError:
     from redis.client import Pipeline
 
+from prometheus_client import Counter
+from prometheus_client import Gauge
+from prometheus_client import Histogram
+
 from baseplate import Span
 from baseplate.clients import ContextFactory
 from baseplate.lib import config
 from baseplate.lib import message_queue
 from baseplate.lib import metrics
 
+from baseplate.lib.prometheus_metrics import default_latency_buckets
+
+PROM_NAMESPACE = "redis"
+PROM_SHARED_LABELS = [
+    f"{PROM_NAMESPACE}_command",
+    f"{PROM_NAMESPACE}_database",
+    f"{PROM_NAMESPACE}_startup_host",
```

Comment:
IRRC, `startup_host` is only a concept in clustered redis, I believe this client is only non-clustered redis. The spec indicates that non-clustered redis should have the label `redis_host`.